ext/net/net.rs

```diff
@@ -0,0 +1,79 @@
+use deno_core::anyhow::Context;
+use deno_core::error::AnyError;
+use fqdn::FQDN;
+use std::net::Ipv4Addr;
+use std::net::Ipv6Addr;
+use std::str::FromStr;
+
+#[derive(Clone, Eq, PartialEq, Hash, Debug)]
+pub enum Host {
+  FQDN(FQDN),
+  Ipv4(Ipv4Addr),
+  Ipv6(Ipv6Addr),
+}
+
+impl Host {
+  pub fn from_str(host: &str, origin_host: &str) -> Result<Self, AnyError> {
```

Comment:
Ditto, `clippy` will complain. Suggest to rename it to `from_host_and_origin_host`